.github/workflows/bottle.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         id: tea
         with:
           srcroot: tea.xyz/var/pantry
-          prefix: ${{ github.workspace }}
+          prefix: $GITHUB_WORKSPACE
```

Comment:
how can this work? `$GITHUB_WORKSPACE` is expanded at the shell level. the Github Actions doesn't expand it when parsing the YAML, nor will action.js. Unless I'm mistaken about the GHA expansion.

Reply:
Very possibly it's just expanding to nothing which is the same thing.